frontend/src/pages/projects/screens/detail/storage/AccessModeLabel.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { AccessMode } from '#~/pages/storageClasses/storageEnums';
+
+// Safely convert string to AccessMode
+export const toAccessMode = (mode: string | undefined): AccessMode | undefined => {
+  if (mode) {
+    const validModes = Object.values(AccessMode);
+    return validModes.find((validMode) => validMode === mode);
+  }
+  return undefined;
+};
+
+const ACCESS_MODE_MAP: Record<AccessMode, string> = {
+  [AccessMode.RWO]: 'ReadWriteOnce (RWO)',
+  [AccessMode.RWOP]: 'ReadWriteOncePod (RWOP)',
+  [AccessMode.RWX]: 'ReadWriteMany (RWX)',
+  [AccessMode.ROX]: 'ReadOnlyMany (ROX)',
+} as const;
+
+type Props = {
+  accessModeString: string | undefined;
+};
+
+const AccessModeLabel: React.FC<Props> = ({ accessModeString }) => {
+  const accessMode = toAccessMode(accessModeString);
+  if (accessMode) {
+    return <span className="pf-v5-u-ml-sm">{ACCESS_MODE_MAP[accessMode]}</span>;
+  }
+  return null;
```

Comment:
If there is no `accessMode` found, I prefer that we still show something like `-` instead of an empty cell. (That will unlikely happen because the k8s will prevent them from editing the access mode after creating)